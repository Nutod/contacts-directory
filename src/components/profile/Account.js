import React from "react";
import styled from "styled-components";
import Sidebar from "../elements/Sidebar/Sidebar";
import { Card } from "../elements/Card/Card";
import theme from "../elements/Icons/Colors";

const ContentWrapper = styled.div`
	padding: 5rem;
	display: grid;
	grid-template-columns: 2fr 2fr 1.5fr;
`;

const CardHeading = styled.p`
	color: #84abf1;
	font-size: 1.3rem;
`;

const CardHeadingSecondary = styled(CardHeading)`
	color: #333;
	font-size: 1.2rem;
	text-align: center;
`;

const ListText = styled.p`
	color: ${theme.lightGreyThree};
	font-size: 1.2rem;
`;

const ListTextSecondary = styled(ListText)`
	color: ${theme.brand};
`;

export default function Account() {
	return (
		<div style={{ display: "flex" }}>
			<Sidebar />
			<ContentWrapper>
				<Card style={{ height: "40rem", width: "30rem" }}>
					<CardHeading>Accounts</CardHeading>
					<div style={{ padding: "2rem" }}>
						<ListText>Import from Device</ListText>
						<ListText>Import from Google</ListText>
						<ListText>Import from Yahoo</ListText>
						<ListText>Import from Facebook</ListText>
						<ListText>Import from Whatsapp</ListText>
					</div>
				</Card>
				<Card style={{ height: "20rem", width: "30rem" }}>
					<CardHeadingSecondary>
						Control how your contacts are organised
					</CardHeadingSecondary>
					<div style={{ padding: "0 2rem" }}>
						<ListTextSecondary>New Contact</ListTextSecondary>
						<ListTextSecondary>Add Account</ListTextSecondary>
						<ListTextSecondary>Sync Options</ListTextSecondary>
					</div>
				</Card>
				<Card style={{ height: "20rem", width: "30rem" }}>
					<CardHeadingSecondary>Manage Profile</CardHeadingSecondary>
					<div style={{ padding: "0 2rem" }}>
						<ListTextSecondary>Edit Profile</ListTextSecondary>
					</div>
				</Card>
			</ContentWrapper>
		</div>
	);
}
