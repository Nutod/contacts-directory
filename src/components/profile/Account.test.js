import React from "react";
import Enzyme, { shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import toJson from "enzyme-to-json";
import Account from './Account'

Enzyme.configure({ adapter: new Adapter() });

describe("<Account />", () => {
	const wrapper = shallow(<Account />);
	it("Should return default true", () => {
		expect(true).toBe(true);
	});

	it("Should match snapshot", () => {
		expect(toJson(wrapper)).toMatchSnapshot();
	});
});
