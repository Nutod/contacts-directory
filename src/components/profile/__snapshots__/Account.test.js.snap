// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Account /> Should match snapshot 1`] = `
<Redirect
  push={false}
  to="/auth/sign-up"
/>
`;
